cmd/dex-worker/main.go

```diff
@@ -48,6 +48,7 @@ func main() {
 
 	enableRegistration := fs.Bool("enable-registration", false, "Allows users to self-register")
 	enableClientRegistration := fs.Bool("enable-client-registration", false, "Allow dynamic registration of clients")
+	registerOnFirstLogin := fs.Bool("register-on-first-login", false, "When a user logs in through a federated identity service, automatically register them if they don't have an account. This flag is independent of enable-registration.")
```

Comment:
On the other hand, I'm going to advocate that we make these exclusive. There's not a good use cases for having both turned on.